devices/cpu/formatIII.c

```diff
@@ -33,7 +33,7 @@ void decode_formatIII(Emulator *emu, uint16_t instruction, bool disassemble)
   Debugger *debugger = emu->debugger;
 
   uint8_t condition = (instruction & 0x1C00) >> 10;
-  int16_t signed_offset = (instruction & 0x03FF) * 2;
```

Comment:
This looks dangerous. Could you explain the change?

Reply:
Ou, I see. The MSP430 ignores lowest bit of address. Ok.

Reply:
Yup, this is a bugfix.